contrib/observability_ready_util/src/main/java/io/opencensus/contrib/observability/ready/util/BasicSetup.java

```diff
@@ -0,0 +1,113 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.observability.ready.util;
+
+import io.opencensus.common.ExperimentalApi;
+import io.opencensus.contrib.grpc.metrics.RpcViews;
+import io.opencensus.exporter.metrics.ocagent.OcAgentMetricsExporter;
+import io.opencensus.exporter.metrics.ocagent.OcAgentMetricsExporterConfiguration;
+import io.opencensus.exporter.trace.ocagent.OcAgentTraceExporter;
+import io.opencensus.exporter.trace.ocagent.OcAgentTraceExporterConfiguration;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.config.TraceConfig;
+import io.opencensus.trace.config.TraceParams;
+import io.opencensus.trace.samplers.Samplers;
+
+/**
+ * Setup class to enable OpenCensus stats and metrics collections easily.
+ *
+ * @since 0.25
+ */
+@ExperimentalApi
+public final class BasicSetup {
+  private static final double DEAFULT_SAMPLING_RATE = 0.0001;
+  private static final String DEAFULT_ENDPOINT = "localhost:55678";
+
+  private BasicSetup() {}
+
+  /**
+   * Enables OpenCensus metric and traces.
+   *
+   * <p>This will register all basic {@link io.opencensus.stats.View}s. When coupled with an agent,
+   * it allows users to monitor application behavior.
+   *
+   * <p>Please note that in addition to calling this method, the application must:
+   *
+   * <ul>
+   *   <li>Include opencensus-contrib-observability-ready-util dependency on the classpath
+   *   <li>Deploy the OpenCensus agent
+   * </ul>
+   *
+   * <p>Example usage for maven:
+   *
+   * <pre>{@code
+   * <dependency>
+   *   <groupId>io.opencensus</groupId>
+   *   <artifactId>opencensus-contrib-observability-ready-util</artifactId>
+   *   <version>${opencensus.version}</version>
+   * </dependency>
+   * }</pre>
+   *
+   * <p>It is recommended to call this method before doing any RPC call to avoid missing stats.
+   *
+   * <pre>{@code
+   * BasicSetup.enableOpenCensus("with-service-name");
+   * }</pre>
+   *
+   * @param serviceName the service name.
+   * @param endPoint the end point of OC-Agent.
+   * @param probability the desired probability of sampling. Must be within [0.0, 1.0].
+   * @since 0.25
+   */
+  public static void enableOpenCensus(String serviceName, String endPoint, double probability) {
+    // register basic rpc views
+    RpcViews.registerAllGrpcBasicViews();
+
+    // set sampling rate
+    TraceConfig traceConfig = Tracing.getTraceConfig();
+    TraceParams activeTraceParams = traceConfig.getActiveTraceParams();
+    traceConfig.updateActiveTraceParams(
+        activeTraceParams.toBuilder().setSampler(Samplers.probabilitySampler(probability)).build());
+
+    // create and register Trace Agent Exporter
+    OcAgentTraceExporter.createAndRegister(
+        OcAgentTraceExporterConfiguration.builder()
+            .setEndPoint(endPoint)
+            .setServiceName(serviceName)
+            .setUseInsecure(true)
```

Comment:
For production deployment Insecure will not work in most if not all cases.

Reply:
Is it ok to assign it `false` (by default) or provide configurable param to control that, What do you suggest?